packages/aws-cdk-lib/aws-events/lib/event-bus.ts

```diff
@@ -309,7 +309,10 @@ export class EventBus extends EventBusBase {
    */
   public readonly eventSourceName?: string;
 
-  private policy?: EventBusPolicy;
+  /**
+   * The EventBusPolicies attached to this event bus
+   */
+  public readonly policies: EventBusPolicy[];
```

Comment:
```suggestion
```
I think the variable should be removed since it is not used at the moment.

Reply:
Not by the lib, but since we are transforming policy statements into event bus policies behind the scenes it seems considerate to provide an accessor for the result. Happy to remove it if you believe strongly otherwise.

Reply:
The pre-existing `policy` variable was `private` and used only to avoid creating duplicate policies.
I don't think it is a good idea to expose a variable that could potentially be changed/removed in the future (considering the note above) and it currently serves no purpose.
The only reason I would keep it is to check for constraints on policies applied on an event bus ([should not exceed 10KB](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-events-eventbuspolicy.html#:~:text=The%20permission%20policy%20on%20the%20event%20bus%20cannot%20exceed%2010%20KB%20in%20size.)), but still, I would keep it `private`.
I don't think that this validation is necessary to be honest, so I would just remove the variable.